DataFormats/DetId/interface/DetId.h

```diff
@@ -23,7 +23,7 @@ class DetId {
   static const int kSubdetOffset       = 25;
 
 
-  enum Detector { Tracker=1,Muon=2,Ecal=3,Hcal=4,Calo=5,Forward=6 };
+  enum Detector { Tracker=1,Muon=2,Ecal=3,Hcal=4,Calo=5,Forward=6, Trigger = 7 };
```

Comment:
@lgray , the problem is in the forward detector (here)[https://github.com/PFCal-dev/cmssw/blob/hgc_trigger_fwk_SLHC26p3/DataFormats/ForwardDetId/interface/ForwardSubdetector.h#L4] where *all* 7 slots are taken. There are three possibilities:
1. Remove one other Forward subdetector
2. Implement this as on other detector
3. not be compliant with the implementations in DetId.h